libwebrtc/Cargo.toml

```diff
@@ -3,24 +3,22 @@ name = "libwebrtc"
 version = "0.1.2"
 edition = "2018"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
-tokio = { version = "1.9.0", features = ["full"] }
-libwebrtc-sys = { path = "../libwebrtc-sys" }
-media_pipeline = { path = "../media_pipeline"}
 bytes = "1.1.0"
+crossbeam-channel = "0.5.1"
 cxx = "1.0.56"
-thiserror = "1.0.30"
-log = "0.4.14"
+dashmap = "4.0.2"
+glob = "0.3.0"
 lazy_static = "1.4.0"
+libwebrtc-sys = { path = "../libwebrtc-sys" }
+log = "0.4.14"
+media_pipeline = { git = "https://github.com/arcas-io/media-pipeline", branch = "main" }
```

Comment:
I assume we'll want to pin to explicit releases/tags in time.

Reply:
Definitely, we'll tag/release those repos in the next pass.